homework-16/common/src/test.rs

```diff
@@ -0,0 +1,34 @@
+#[cfg(test)]
+mod tests {
+    use crate::cli;
+    use crate::util::{base64_decode, base64_encode};
+    use cli::{parse_args, CliArg, HOST_DEFAULT, PORT_DEFAULT};
+
+    // Test CLI parsing
+    #[test]
+    fn given_no_parameters_specified_when_cli_parsed_then_defaults_returned() {
```

Comment:
longest test names of all students, but I see what you are doing, hahaha, very good